applications/processor/aggregator-processor/README.adoc

```diff
@@ -21,12 +21,12 @@ Properties grouped by prefix:
 
 === aggregator
 
-$$aggregation$$:: $$SpEL expression for aggregation strategy. Default is collection of payloads.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$correlation$$:: $$SpEL expression for correlation key. Default to correlationId header.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$group-timeout$$:: $$SpEL expression for timeout to expiring uncompleted groups.$$ *($$Expression$$, default: `$$<none>$$`)*
-$$message-store-entity$$:: $$Persistence message store entity: table prefix in RDBMS, collection name in MongoDb, etc.$$ *($$String$$, default: `$$<none>$$`)*
-$$message-store-type$$:: $$Message store type.$$ *($$String$$, default: `$$<none>$$`)*
-$$release$$:: $$SpEL expression for release strategy. Default is based on the sequenceSize header.$$ *($$Expression$$, default: `$$<none>$$`)*
+$$aggregation$$:: $$<documentation missing>$$ *($$Expression$$, default: `$$<none>$$`)*
```

Comment:
If you run `./quick-compile.sh` it will update these readme as the content is currently missing. 

<img width="1671" alt="Screenshot 2024-03-08 at 09 52 09" src="https://github.com/spring-cloud/stream-applications/assets/28907971/a5fbbfd6-bbf3-45c2-be2c-01c1822c5099">